src/vs/base/common/async.ts

```diff
@@ -166,26 +166,36 @@ export interface ITask<T> {
  * 			throttler.queue(deliver);
  * 		}
  */
-export class Throttler {
+export class Throttler implements IDisposable {
```

Comment:
Throttler is used in ThrottlerDelayer in the same file. Can you update that as well to call `dispose()`

Reply:
https://github.com/microsoft/vscode/issues/185689